packages/contracts-periphery/contracts/foundry-tests/citizen-house/BadgeAdmin.t.sol

```diff
@@ -0,0 +1,692 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.15;
+
+import { BadgeAdmin } from "../../universal/citizen-house/BadgeAdmin.sol";
+import { Badge } from "../../universal/citizen-house/Badge.sol";
+import { Test, stdError } from "forge-std/Test.sol";
+
+function mkadr(string memory name) pure returns (address) {
+    address adr = address(uint160(uint256(keccak256(abi.encodePacked(name)))));
+    return adr;
+}
+
+contract TBadge is Badge {
+    constructor() Badge("example", "ex", "example.com") {}
+}
+
+contract TBadgeAdmin is BadgeAdmin {
```

Comment:
I think you should get rid of this test wrapper contract and just set the contract up in the `BadgeAdminTest` contract, either in the constructor or `setUp`